easy-ui-react/src/SearchNav/LogoGroup.tsx

```diff
@@ -13,23 +12,23 @@ export type LogoGroupProps = {
   children: ReactNode;
 };
 
-export function LogoGroup(props: LogoGroupProps) {
-  const { children } = props;
-
-  const items = useMemo(() => {
-    return flattenChildren(children);
-  }, [children]);
-
-  const logo = items[0];
-  const select = items.length === 2 ? items[1] : null;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export function LogoGroup(_props: LogoGroupProps) {
+  const { logo, emphasizedText, selector } = useInternalSearchNavContext();
```

Comment:
**thought**: this is an interesting pattern. it might be worth leaving a short comment above the eslint-disable line in these situations that says something along the lines of "we derive the props from context instead..." etc